modules/ROOT/pages/composite-databases/index.adoc

```diff
@@ -1,11 +1,88 @@
 [role=enterprise-edition]
 [[composite-databases]]
 = Composite databases
-:description: This section describes administration and operation of composite databases. 
+:description: An introduction to Composite databases.
 
-* xref:composite-databases/introduction.adoc[Introduction]
-* xref:composite-databases/queries.adoc[Queries]
-* xref:composite-databases/considerations.adoc[Further considerations]
-* xref:composite-databases/sharding-with-copy.adoc[Sharding data with the `copy` command]
+[[composite-databases-introduction]]
+== Introduction
 
+A Composite database is a special type of database introduced in Neo4j 5.
+It supersedes the implementation of the previous Fabric in Neo4j 4.
 
+Fabric as a concept describes how different parts of a domain (graph data and computing resources), which can be in the same or multiple Neo4j DBMSs, can work as a unified system with a single access point.
+Composite databases are the means to access this partitioned data or graphs with a single Cypher query.
+
+Composite databases *do not store data* independently.
+They use _database aliases_, which can be local, remote, or sharded.
```

Comment:
What is "sharded" supposed to mean here? There are only two kinds of aliases atm